app/move/fields/create.js

```diff
@@ -38,6 +40,67 @@ function assessmentCategory(category) {
   }
 }
 
+function explicitYesNo(name) {
+  return {
+    validate: 'required',
+    component: 'govukRadios',
+    name: name,
+    fieldset: {
+      legend: {
+        text: `fields::${name}.label`,
+        classes: 'govuk-fieldset__legend--m',
+      },
+    },
+    items: [
+      {
+        value: 'yes',
+        text: 'Yes',
+      },
+      {
+        value: 'no',
+        text: 'No',
+      },
+    ],
+  }
+}
+
+function appendDependent(questions, assessmentCategory, field, key) {
```

Comment:
This method and `decorateWithExplicitFields` should probably live in the [field helpers](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/common/helpers/field.js) file.

They're more helper methods than field definitions related specifically to the create journey.